packages/liferay-npm-scripts/src/scripts/lint.js

```diff
@@ -238,26 +263,131 @@ function formatter(results) {
 	return output;
 }
 
+function isSCSS(filePath) {
+	return hasExtension(filePath, EXTENSIONS.scss);
+}
+
+async function lintSCSS(source, onReport, options = {}) {
+	const {filePath, fix, quiet} = options;
+
+	const messages = [];
+
+	let errorCount = 0;
+	let warningCount = 0;
+
+	// These two are here for symmetry with ESLint; however, stylelint
+	// doesn't tell us which problems are autofixable and there is no
+	// totally reliable way to guess, so these counts will always be
+	// (possibly innaccurately) zero.
+	const fixableErrorCount = 0;
+	const fixableWarningCount = 0;
+
+	const {output, results} = await stylelint.lint({
+		code: source,
+		codeFilename: filePath,
+		config: {
+			rules: {
```

Comment:
Isn't there a way to provide the rules via config as we do for the other tools?

Reply:
There [are `.stylelintrc` (etc) files](https://stylelint.io/user-guide/configuration), although I have not tested to see whether the Node.js API reads them automatically. In any case, I was thinking of sticking the defaults [in here with all the others](https://github.com/liferay/liferay-npm-tools/tree/master/packages/liferay-npm-scripts/src/config).